test/e2e/package.json

```diff
@@ -18,7 +18,9 @@
 		"clean": "yarn clean:packages && yarn clean:output && rm -rf ./cookies && rm -rf ./node_modules",
 		"clean:packages": "yarn workspace @automattic/jest-circus-allure-reporter clean && yarn workspace @automattic/calypso-e2e clean",
 		"clean:output": "rm -rf ./allure-results && rm -rf ./results",
-		"build": "yarn workspace @automattic/jest-circus-allure-reporter build && yarn workspace @automattic/calypso-e2e build"
+		"build": "yarn workspace @automattic/jest-circus-allure-reporter build && yarn workspace @automattic/calypso-e2e build",
+		"start": "yarn jest",
+		"start:debug": "PWDEBUG=1 DEBUG=pw:api yarn run start"
```

Comment:
`yarn start <spec_path` doesn't actually save any keystrokes, I just realized.

Perhaps only `yarn debug` is necessary for now?

Reply:
**thought:** I prefer `yarn start <spec_path>` despite more keystrokes because it's "easier" to remember what it tries to do. 

`yarn debug` is also preferable. 👍🏻

Reply:
I was thinking in bed before sleeping and think maybe `yarn test` is also a viable alternative.

`yarn test` and either `yarn test:debug` or `yarn debug`.

Reply:
Changed to `yarn test` and `yarn debug` in b6b9f48696703a823482b4b03d505bcebeb85399.